db/migrations/20230713155553_endpoint_versions.sql

```diff
@@ -0,0 +1,67 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE unweave.endpoint_eval DROP CONSTRAINT endpoint_eval_endpoint_id_fkey;
+ALTER TABLE unweave.endpoint_check DROP CONSTRAINT endpoint_check_endpoint_id_fkey;
+
+ALTER TABLE unweave.endpoint_check_step DROP CONSTRAINT endpoint_check_step_check_id_fkey;
+
+TRUNCATE unweave.endpoint_eval;
+TRUNCATE unweave.endpoint_check;
+
+ALTER TABLE unweave.endpoint_check_step 
+ADD CONSTRAINT endpoint_check_step_check_id_fkey 
+FOREIGN KEY (check_id) 
+REFERENCES unweave.endpoint_check (id);
+
+DROP TABLE unweave.endpoint;
+
+CREATE TABLE unweave.endpoint (
+    id text NOT NULL PRIMARY KEY,
```

Comment:
Could we insert an icon in here while we're at it? We need it on the frontend

`
CREATE TABLE unweave.endpoint (
    id text NOT NULL PRIMARY KEY,
    icon text not null default 🚀
...
`